x-pack/plugins/security_solution/public/common/components/super_date_picker/index.test.tsx

```diff
@@ -17,7 +17,7 @@ import {
   kibanaObservable,
   createSecuritySolutionStorageMock,
 } from '../../mock';
-import { createUseUiSetting$Mock } from '../../mock/kibana_react';
+import { createUseUiSetting$Mock } from '../../lib/kibana/kibana_react.mock';
```

Comment:
Nice cleanup! Love how the mock files pattern is paying off more and more and how it gets easier to see where we can remove ours for theirs and reduce code in the code base.